docs/specs/tripgo.swagger.yaml

```diff
@@ -1768,9 +1770,23 @@ definitions:
       openingHours:
         $ref: '#/definitions/OpeningHours'
       pricingTable:
-        $ref: '#/definitions/PricingTable'
+        type: array
+        items:
+          $ref: '#/definitions/PricingTable'
+      restrictions:
+        description: List of car park restrictions.
```

Comment:
Can you provide some more information on the restrictions? Is this a localised text that can be presented as is to users? Why is it a list and not just a single long string? (Like we have in `info`)